### ISSUE TITLE
LilyGo T-SIM7080G board damaged by humidity ?

### ISSUE THREAD

**Issue by mathieucarbou** [Xinyuan-LilyGO/LilyGo-T-SIM7080G#62]
Hello,

I am testing the lilygo board outside since a few days. There is a lot of humidity outside, the lilygo is in a waterproof box (but the box has a hole for the wires on the side, so the air can enter but not the rain).

I am wondering what's this on the board ? it was not there before. It feels like the humidity stats damaging the board ?

![PXL_20240425_170734139 NIGHT](https://github.com/Xinyuan-LilyGO/LilyGo-T-SIM7080G/assets/61346/f34d19f9-03b0-4894-8441-4e37001d4e8f)

The connected usb-c device is this solar panel : https://fr.aliexpress.com/item/1005005509831452.html

* Tension de fonctionnement: 5 ~ 5.5V 
* Courant de travail: 0.8A ~ 1A(Max.)

**Comment by lewisxhe** [comment]
Obviously it is affected by moisture. You can remove the damaged original parts first, then power on and test whether they can work.

**Comment by mathieucarbou** [comment]
> Obviously it is affected by moisture. You can remove the damaged original parts first, then power on and test whether they can work.

I don't have the tools for such precise work at home. Right now, I can power the board with USB-C, solar panel connector does no work, and I can power through internal battery.

But I found that when I have a USB-C solar panel connected and internal battery, when the solar panel is not able to produce enough current, the device shuts down instead of switching to the battery. 

I do not have this behaviour with the second board.

I have ordered a spray for electronic boards... 

But what I don't understand is why these boards come without any varnish, silicon or resin to protect the components from humidity / moisture ?

Also, is there a solution I could do to cleanup without having to remove the components ?

How I could check if some are damaged ?
